src/main/java/gameframework/motion/blocking/MoveBlockerCheckerDefaultImpl.java

```diff
@@ -10,36 +10,60 @@
 import java.util.Vector;
 import java.util.concurrent.ConcurrentLinkedQueue;
 
+/**
+ * Default implementation of the MoveBLockerChecker interface. For more informations,
+ * please refer to the MoveBlockerChecker interface.
+ */
 public class MoveBlockerCheckerDefaultImpl implements MoveBlockerChecker {
-	private ConcurrentLinkedQueue<MoveBlocker> moveBlockers;
-	private MoveBlockerRulesApplier moveBlockerRuleApplier;
+	/**
+	 * A queue containing all the moveBlockers to check for when verifying if a
+	 * GameMovable can move
+	 */
+	protected final ConcurrentLinkedQueue<MoveBlocker> moveBlockers;
+
+	/**
+	 * The rule applier used when checking if a GameMovable can move
+	 */
+	protected MoveBlockerRulesApplier moveBlockerRuleApplier;
 
 	public MoveBlockerCheckerDefaultImpl() {
 		moveBlockers = new ConcurrentLinkedQueue<MoveBlocker>();
 		this.moveBlockerRuleApplier = new MoveBlockerRulesApplierDefaultImpl();
 	}
 
+	/**
+	 * @see gameframework.motion.blocking.MoveBlockerChecker#addMoveBlocker(gameframework.motion.blocking.MoveBlocker)
+	 */
 	@Override
-	public void addMoveBlocker(MoveBlocker p) {
+	public void addMoveBlocker(final MoveBlocker p) {
```

Comment:
adding `final` does not seem relevant to this PR. Please pay attention to only change 1 thing at a time. In a documentation PR, I don't expect to see code changes.